greykite/sklearn/estimator/auto_arima_estimator.py

```diff
@@ -317,6 +317,13 @@ def predict(self, X, y=None):
             return_conf_int=self.return_conf_int,
             alpha=(1-self.coverage)
         )
+        # Starting pmdarima=2.0.0, pmdarima.utils.check_endog added parameter preserve_series
+        # with default True, causing prediction[0] to result in pd.Series instead of ndarray
+        if type(predictions[0]) == pd.Series:
+            # Tuple can't be changed - convert to list for a change
+            pred_list = list(predictions)
+            pred_list[0] = pred_list[0].to_numpy()
+            predictions = tuple(pred_list)
```

Comment:
Celá tato operace by se obešla i bez konverze na list: je možné rovnou vytvořit nový tuple z toho původního:

```
predictions = (predictions[0].to_numpy(), *predictions[1:])
```

Operátor `*` "rozbalí" zadaný argument, slice `predictions[1:]` zase vrátí všechy prvky kromě toho prvního, který transformujeme.

Reply:
Wow, moc pěkné, to si schovám.